assets/src/scripts/monitoring-location/components/hydrograph/parameters.js

```diff
@@ -248,10 +281,17 @@ export const drawSelectionList = function(container, store, siteno) {
             .attr('id', `expansion-container-row-${parameter.parameterCode}`)
             .attr('class', 'expansion-container-row')
             .attr('hidden', 'true');
+
         // Add the rows nested in the expansion container
         if (parameter.waterAlert.hasWaterAlert) {
             drawWaterAlertRow(expansionContainerRow, siteno, parameter);
         }
+
+        // Add the sampling method selection list to the parameter loaded when the page is first opened
```

Comment:
nit: Do you mean "row is first opened"?

Reply:
Well, kinda. I added a bit more explanation, not sure if it helps. This code deals with the fringe case when the page loads and the default parameter has methods, but the user never actually clicks the row, but uses the row expansion toggle (simple no? :) ) .       

This is what I added:
 // Add the sampling method selection list to the parameter loaded when the page is first opened if it has methods.
 // This allows the user to use the row expansion toggle to see the methods when the row has never been physically clicked.

Reply:
Okay. There does seem to be some "magic" around all of the clicking/expanded/contracting. I'm afraid that this code will be somewhat brittle. Hopefully I am wrong.

Reply:
True. If we had all of the sampling method options for every parameter when the table was created things would be less complicated. So that might be a motivating factor to making a change to the data reducer as some future point.